pkg/pillar/Dockerfile.in

```diff
@@ -49,7 +49,7 @@ RUN apk add --no-cache \
     apk-cron coreutils dmidecode sudo libbz2 libuuid ipset \
     libaio logrotate pixman glib curl radvd perl ethtool \
     util-linux e2fsprogs libcrypto1.0 xorriso \
-    python libpcap libffi jq
+    python libpcap libffi jq e2fsprogs-extra keyutils
```

Comment:
What's the size impact of adding these?

Reply:
e2fsprogs-extra - 608kb
keyutils - 116kb